The test failing in `trunk / test-coreml-model / macos-job (push) ` seems legit. Could you help fixing it?
```
+ python -m examples.apple.coreml.scripts.export --model_name=mobilebert
  Torch version 2.5.0.dev20240901 has not been tested with coremltools. You may run into unexpected errors. Torch 2.3.0 is the most recent version that has been tested.
  Traceback (most recent call last):
    File "<frozen runpy>", line 198, in _run_module_as_main
    File "<frozen runpy>", line 88, in _run_code
    File "/Users/ec2-user/runner/_work/executorch/executorch/pytorch/executorch/examples/apple/coreml/scripts/export.py", line 160, in <module>
      model, example_inputs, _ = EagerModelFactory.create_model(
                                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    File "/Users/ec2-user/runner/_work/executorch/executorch/pytorch/executorch/examples/models/model_factory.py", line 38, in create_model
      module = importlib.import_module(
               ^^^^^^^^^^^^^^^^^^^^^^^^
    File "/Users/ec2-user/runner/_work/_temp/conda_environment_10762964118/lib/python3.11/importlib/__init__.py", line 126, in import_module
      return _bootstrap._gcd_import(name[level:], package, level)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    File "<frozen importlib._bootstrap>", line 1204, in _gcd_import
    File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
    File "<frozen importlib._bootstrap>", line 1147, in _find_and_load_unlocked
    File "<frozen importlib._bootstrap>", line 690, in _load_unlocked
    File "<frozen importlib._bootstrap_external>", line 940, in exec_module
    File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
    File "/Users/ec2-user/runner/_work/executorch/executorch/pytorch/executorch/examples/models/mobilebert/__init__.py", line 7, in <module>
      from .model import MobileBertModelExample
    File "/Users/ec2-user/runner/_work/executorch/executorch/pytorch/executorch/examples/models/mobilebert/model.py", line 11, in <module>
      from transformers import AutoTokenizer, MobileBertModel  # @manual
      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

--- a/backends/apple/coreml/scripts/install_requirements.sh
+++ b/backends/apple/coreml/scripts/install_requirements.sh
@@ -47,6 +47,11 @@ cmake --build "$COREMLTOOLS_DIR_PATH/build" --parallel
 
 echo "${green}ExecuTorch: Installing coremltools."
 pip install "$COREMLTOOLS_DIR_PATH"
+# CoreMLTools have started supporting numpy 2.0,
+# but ExecuTorch example model test env is still using older transformers,
+# so for now we will need to downgrade numpy to 1.x
+# TODO: Remove this numpy downgrade once later transformers starts to be used
+pip install numpy==1.26.4
 STATUS=$?
 if [ $STATUS -ne 0 ]; then
     echo "${red}ExecuTorch: Failed to install coremltools."

